docs/EditContext/explainer.md

```diff
@@ -188,9 +189,81 @@ interface EditContext : EventTarget {
     attribute EventHandler oncompositionend;
 };
 ```
+## Difference between DIV with Contenteditable and DIV with EditContext
```

Comment:
Why <div> specifically? Why not any contenteditable element?

Reply:
I use <div> just to give a more concrete example so the audience of the explainer can relate and understand the concept more easily. 

(Discussed offline and decided to update the title.)